docs/device-management/5-ota/5-migrating-from-releases/README.md

```diff
@@ -0,0 +1,139 @@
+we recommend transitioning to Cohorts as soon as possible.
+
+## Differences between Releases and Cohorts
+
+Cohorts bring two fundamental upgrades to the way OTA Updates are distributed
+from your devices. The most important change is that you now have to explicitly
+add a device to a Cohort, instead of relying on the blueprint and tags matching
+mechanism in Releases and Artifacts. The second fundamental change is the way
+each update is rolled out — with Releases, each Release could be enabled and
+disabled at any time, and there could be multiple Releases enabled at the same
+time. With Cohorts, updates (referred to as Deployments) are rolled out in a
+linear timeline of events, with no option for retracting existing Deployments.
+
+Let's take a closer look at how this affects your workflow.
+
+### Explicit targeting
+
+Releases targeted devices through implicit matching of tags and blueprints. When
+a Release was rolled out, the Golioth Cloud would determine which devices should
+be receiving the Release based on the tags and blueprints of each device in the
+project. While this made it easy to target new devices by changing their tags,
+it meant that actions like adding a tag to a device carried an unproportionate
+weight to it, as this could make the device pull down a new OTA update
+unintentionally, which could be catastrophic. Additionally, there wasn't a clear
+indication about which devices would receive a new Release, which made the issue
+worse.
+
+The new workflow alleviates this problem by enforcing explicit targeting. Every
+device that needs to receive an OTA update now has to be assigned to a Cohort
+first. While this adds an extra step to update management, it makes it a lot
+harder to push the wrong OTA update by mistake.
+
+### Linear timeline of Deployments
+
+Releases featured a roll out toggle that would let you enable or disable
+releases at any time. This caused Releases to behave more like firmware
+configurations you could move between than updates you could push to your fleet.
+
+In the new workflow, Deployments are immutable events. You can still move
+between different versions of your firmware, but to roll back to a previous
+version, you now have to issue a new Deployment that instructs the device to go
+back. A Deployment is applied to every device in the Cohort, and there's only
+ever one active Deployment for each Cohort. This makes it easier to determine
+exactly what firmware each device is expected to have — not just right now, but
+also historically.
+
+### Other differences
+
+In addition to explicit targeting and a linear timeline of Deployments, there
+are a couple of smaller differences between Releases and Cohorts:
+
+- **Adding a device to a Cohort will push a manifest update to the device right
+  away.** With Releases, devices would only receive an update notification when
+  you rolled out a Release, but could still pull for the active Release between
```

Comment:
poll may not be quite right here, but wasn't sure if you had intended to use it or not?
```suggestion
  you rolled out a Release, but could still poll for the active Release between
```